sunpy/net/hek/hek.py

```diff
@@ -59,7 +59,8 @@ def _download(self, data):
             data['page'] = page
             fd = urllib.request.urlopen(self.url+urllib.parse.urlencode(data))
             try:
-                result = json.load(fd)
+                result = codecs.decode(fd.read(), encoding='utf-8', errors='ignore')
```

Comment:
Does it work with `'replace'` here instead of `'ignore'`? If it did some information on the character would be preserved.

Reply:
Let me try it.

Reply:
I replaced it and the test still passes.